pandas_gbq/gbq.py

```diff
@@ -831,7 +837,10 @@ def to_gbq(dataframe, destination_table, project_id, chunksize=10000,
     table = _Table(project_id, dataset_id, reauth=reauth,
                    private_key=private_key)
 
-    table_schema = _generate_bq_schema(dataframe)
+    if not table_schema:
+        table_schema = _generate_bq_schema(dataframe)
+    else:
+        table_schema = dict(fields=table_schema)
```

Comment:
Current implementation will throw `StreamingInsetError` after a chunk is done (see tests) along printing error trace from the BQ API. Which is OK.